pytest_pootle/fixtures/models/permission.py

```diff
@@ -54,3 +54,10 @@ def administrate(pootle_content_type):
     """Require the `suggest` permission."""
     return _require_permission('administrate', 'Can administrate a TP',
                                pootle_content_type)
+
+
+@pytest.fixture(scope="session")
+def translate(pootle_content_type):
```

Comment:
not sure if i understand, but either its created in env, in which case make this a test-scoped fixture, or it should be created in env